Tests/ApolloTests/CacheKeyForFieldTests.swift

```diff
@@ -48,8 +68,8 @@ class CacheKeyForFieldTests: XCTestCase {
   }
   
   func testFieldWithInputObjectArgumentIsOrderIndependent() {
-    let field1 = GraphQLField("hero", arguments: ["episode": Episode.jedi, "nested": ["foo": "a", "bar": "b"]], type: .scalar(String.self))
-    let field2 = GraphQLField("hero", arguments: ["episode": Episode.jedi, "nested": ["bar": "b", "foo": "a"]], type: .scalar(String.self))
+    let field1 = GraphQLField("hero", arguments: ["episode": "JEDI", "nested": ["foo": "a", "bar": "b"]], type: .scalar(String.self))
```

Comment:
Why are we not able to use the enum here anymore?

Reply:
Well the reason I caught it is because this is being converted into an `InputValue` and there is no `ExpressibleByEnumWithRawValueLiteral` in the swift language haha. 

But this is actually better because that is what the generated code that the current Codegen outputs. We want these tests to reflect the same thing that the generated code looks like, so this is better anyways.

Reply:
I think you could do an extension on  `RawRepresentable where RawValue == String` or something like that.

I'm also slightly confused about where this gets turned from the enum to the raw string value - we can generate it with the string for things specified at compile time, but not runtime.

Reply:
For the new version of Codegen, I fully intend on improving this. The current codegen already uses the Raw String here, and to support using an enum, I'd need the codegen to wrap the enum in an `InputValue` case. Something like `"episode": .enum(Episode.jedi)`. I haven't decided the exact format for that yet, but since the old codegen already does it this way, I don't have to worry about that just yet.

Reply:
I'm slightly missing where we're doing the conversion to a string if the enum value is passed in at runtime - the query you added specifies it has to be a particular episode, but what about something like `CreateReviewForEpisodeMutation` where the `variables` is using a `GraphQLMap` with the enum value?

Reply:
Great question. In `InputValues` we have this case for evaluating variables. 

```swift
case .variable(let name):
      guard let value = variables?[name] else {
        throw GraphQLError("Variable \(name) was not provided.")
      }
      return value.jsonValue
```

The `GraphQLMap` is converted to it's `jsonValue` in that last line. The `jsonValue` is using the `Dictionary` `jsonValue` extension in `JSONStandardTypeConversions` to recursively convert all of its values!

Reply:
Goooot it, this was happening in `extension GraphQLVariable: GraphQLInputValue {` before. Thank you!